dxr/plugins/python/indexers.py

```diff
@@ -89,6 +199,29 @@ def _all_needles(self):
                 needle_type = self.needle_types[node.__class__]
                 yield self._needle(needle_type, node.name, node.start, node.end)
 
+            # Index the class hierarchy for classes for the derived: and
+            # bases: filters.
+            if isinstance(node, ast.ClassDef):
+                class_name = module_name + '.' + node.name
+
+                # Index both the full absolute base name and the
+                # local name for conveniece.
+                bases = self.get_bases_for_class(class_name)
+                for base in bases:
+                    base_name = base.split('.')[-1]
+                    yield self._needle('py_derived', base, node.start,
```

Comment:
Rather than filing 2 separate needles, have you considered using (and promoting up to dxr.filters) clang.filters._QualifiedNameFilter and using a needle like QUALIFIED_NEEDLE? That way, the `+` syntax for fully qualified names would work, so you could search for `+foo.bar` and find just that rather than also finding "foo.bar.foo.bar".